Library/Homebrew/dev-cmd/tests.rb

```diff
@@ -1,4 +1,4 @@
-#:  * `tests` [`-v`] [`--coverage`] [`--generic`] [`--no-compat`] [`--only=`<test_script>`:`<test_method>] [`--seed` <seed>] [`--trace`] [`--online`] [`--official-cmd-taps`]:
+#:  * `tests` [`-v`] [`--coverage`] [`--generic`] [`--no-compat`] [`--only=`<test_name>] [`--seed` <seed>] [`--online`] [`--official-cmd-taps`]:
```

Comment:
Doesn't seem possible when using `parallel_rspec`. Maybe @alyssais knows more?

A workaround would be to run default `rspec` when only a single test should be run.